lib/rototiller/task/collections/argument_collection.rb

```diff
@@ -0,0 +1,14 @@
+require 'rototiller/task/params/argument'
+require 'rototiller/task/collections/switch_collection'
+
+module Rototiller
+  module Task
+
+    class ArgumentCollection < SwitchCollection
```

Comment:
this needs to be commented just so readers know why we're inheriting from SwitchCollection instead of ParamCollection if they don't know that arguments are really just switches...
but i'm not sure i want to hold up the merge